### PR TITLE
TNO-267: Page size consistency

### PR DESCRIPTION
**Pull request by jdtoombs**
formpage

All that was added to `ContentListView` and `ContentForm` was the addition of wrapping it in the new `<FormPage>`. Indentation is the explanation for all the additions.

### CODE REVIEW

File: app/editor/src/features/content/form/ContentForm.tsx [hunk 1]
```diff
@@ -80,170 +81,173 @@ export const ContentForm: React.FC = () => {
...
-                        width={FieldSize.Medium}
-                        name="otherSource"
-                        label="Other Source"
-                        onChange={(e) => {
-                          const value = e.currentTarget.value;
-                          props.setFieldValue('otherSource', value);
-                          props.setFieldValue('source', value);
-                          if (!!value) {
-                            props.setFieldValue('dataSourceId', undefined);
+    <FormPage height="fit-content">
```
Comment (Fosol) [inline app/editor/src/features/content/form/ContentForm.tsx:84]: This isn't ideal.  Generally it's preferred to always have the styled component as the root object returned.  Either the `styled.ContentForm` should inhert from `FormPage` or it should wrap it.

Comment (Fosol) [reply app/editor/src/features/content/form/ContentForm.tsx:84]: Also trying to understand why your auto formatting is messing with the whole file.  It's impossible to see what has changed.

File: app/editor/src/features/content/list-view/ContentListView.tsx [hunk 2]
```diff
@@ -127,265 +128,267 @@ export const ContentListView: React.FC = () => {
...
-                  }
-                />
-              </div>
-            </div>
-          </div>
-        </div>
-        <div className="box">
-          <h2 className="caps">Advanced Search</h2>
-          <div style={{ display: 'flex', flexDirection: 'row', minWidth: '550px' }}>
+    <FormPage>
```
Comment (Fosol) [inline app/editor/src/features/content/list-view/ContentListView.tsx:131]: The styled component should always be the root.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: app/editor/src/features/content/form/ContentForm.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/features/content/form/ContentForm.tsx]: In this case it was because of the new indentation due to adding a new parent

**Review by Fosol — APPROVED** [review]

**Pull request merged**